src/app/gallery/checkout/page.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+import { useEffect } from 'react';
+import { Button } from '~/components/ui/button';
+import { stripePromise } from './load-stripe';
+
+
+export default function PreviewPage() {
+
+    // eslint-disable-next-line
+    const stripe = stripePromise;
+
+    useEffect(() => {
+        // Check to see if this is a redirect back from Checkout
+        const query = new URLSearchParams(window.location.search);
+        if (query.get('success')) {
```

Comment:
Really good little system! Very basic but seems like it would work

Reply:
Might want to display something on the screen though, as normal users won't check the console for `console.log` statements!